packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -313,9 +313,8 @@ export const Webchat = forwardRef((props, ref) => {
     session = initSession(session)
     updateSession(session)
     if (
-      !devSettings ||
-      Object.keys(devSettings).length === 0 ||
-      devSettings.keepSessionOnReload
+      !initialDevSettings ||
```

Comment:
`!initialDevSettings` let us know that webchatApp is being ran in production environment